app/uart.h

```
@@ -303,40 +303,81 @@ void uart_event_handle(app_uart_evt_t* p_event)
+                                ble_conn_flag = BLE_ON_TEMPO;
+                                NRF_LOG_INFO("RCV ble flag start adv flag.");
+                                break;
+                            case BLE_OFF_TEMPO:
+                                ble_conn_flag = BLE_OFF_TEMPO;
+                                NRF_LOG_INFO("RCV ble flag stop adv flag.");
+                                break;
+                            case BLE_STATUS:
+                                trans_info_flag = RESPONESE_BLE_STATUS;
+                                break;
+                            case BLE_PASSKEY_ACCEPT:
+                                if(waiting_passkey_response && m_conn_handle != BLE_CONN_HANDLE_INVALID)
```

Clang (14.0.6)

[note] +5, including nesting penalty of 4, nesting level increased to 5

(clang)